docs/RESOURCES.md

```diff
@@ -0,0 +1,7 @@
+## Useful Resources
+
+* [StackOverflow](http://stackoverflow.com/questions/tagged/java).
+* [The official java subreddit](https://www.reddit.com/r/java)
+* [The exercism/java Gitter room](https://gitter.im/exercism/java)
```

Comment:
I'd remove this since I think these resources are mostly intended to be external?